### DISCUSSION TITLE
q-table row selection problem

### DISCUSSION THREAD

**Discussion by CrawfordW** [quasarframework/quasar#9414] [General - Components / Directives / etc]
When I click on a row, it gets selected OK but ALL rows are then highlighted.
I've had it working in other components, but can't seem to get it working here.
I must be doing something wrong, but I'm stumped -  help will be appreciated.

Here's a codepen with an example:
https://codepen.io/crawfordw/pen/VwpWawK?editors=1111

**Comment by ontwikkelfabriek** [comment]
@CrawfordW 

you should know ;-)

https://forum.quasar-framework.org/topic/5674/select-q-table-row-programatically/3

hmmm.... does not appear to be working

**Reply by CrawfordW** [reply]
Yes, if I use that method, there is no highlighting at all. But your reminder made me see the problem...
I was using a row-key that was not unique to the row.
So problem solved! Thanks.

**Reply by ontwikkelfabriek** [reply]
@CrawfordW 
here's a working codepen anyways...

https://codepen.io/ontwikkelfabriek/pen/poeweBO

**Reply by CrawfordW** [reply]
Yes thanks for your help.
Now I have to try multiple row selections with shift-click...
Will need to somehow get the index of each row.

**Reply by CrawfordW** [reply]
Got multiple row selection working. Here's the codepen...
https://codepen.io/crawfordw/pen/JjXodbp

**Reply by bagladivyang03** [reply]
@CrawfordW  U can this as well for single row click.
https://codepen.io/smolinari/pen/eYOPZoa?editors=1010

**Reply by ontwikkelfabriek** [reply]
@CrawfordW 

Very nice with the crt+shift options! Will bookmark this one.
